tests/unit/test_notebooks.py

```diff
@@ -111,75 +110,6 @@ def _nb_modify(line):
         _run_notebook(tmpdir, notebook_path, _nb_modify)
 
 
-def test_movielens_example(tmpdir):
```

Comment:
This does lose test coverage of the Movielens example in this repo. Since that example would now rely on multiple Merlin libraries, I'd propose moving it to the top-level Merlin repo if we want to keep it.